src/libstd/collections/hash/map.rs

```diff
@@ -1218,6 +1218,7 @@ impl<K, V, S> Default for HashMap<K, V, S>
     where K: Eq + Hash,
           S: BuildHasher + Default,
 {
+    /// Creates a `HashMap<K, V, S>`, with initial `Default` hasher.
```

Comment:
It can say the HashMap is empty, too. "Creates an empty `HashMap<K, V, S>", with the default value for the hasher"